### DISCUSSION TITLE
Correct Settings for Shutdown

### DISCUSSION THREAD

**Discussion by Jackalolz** [JPersson77/LGTVCompanion#194] [General]
Hi Hive.

I've got an issue with my companion and LG C3.
When I first installed this app it worked perfectly. Then after a few months of use, the following issue started to occur. 

TV Companion works flawlessly for powering up, restarting, sleep modes, and waking from sleep but it does not work with shutting down. Hasn't worked for a few weeks but I think it's annoying me enough to make a github account to voice my problem. 

Television has a dedicated IP address, running the latest version of LGCompanion and it will not communicate shutdown in both late and early modes. 

Any ideas?

**Comment by JPersson77** [comment]
Hi @Jackalolz, 

I have no immediate solution for you but lets try to find one . Can you enable logging please, and run through a system shutdown. Then paste a link to a copy of log.txt and config.json and I will have a look. Both are found in c;/ programdata/lgtv companion/

**Reply by Jackalolz** [reply]
Please find my logged files

I shut down the system, booted the system, and went to idle for examples.

link can view items: https://drive.google.com/drive/folders/1oa75hKQZwxH4H0ha5QSgnEmfArQ6FumM?usp=sharing

**Comment by JPersson77** [comment]
Hey again, you have misconfigured the "restart words" it seems. Please check so it looks like this:
![image](https://github.com/JPersson77/LGTVCompanion/assets/77928210/741c0e3a-e6c4-4fe9-aa76-b64e8cc1753d)

(you can click the "what's this?" link to read more about why it needs to look like this.
